modules/EventTracker.js

```diff
@@ -0,0 +1,187 @@
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['underscore', 'backbone', './ServiceCell'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('underscore'), require('backbone'), require('./ServiceCell'));
+  } else {
+    root.Torso = root.Torso || {};
+    root.Torso.EventTracker = factory(root._, root.Backbone, root.Torso.ServiceCell);
+  }
+}(this, function(_, Backbone, ServiceCell) {
+  'use strict';
+
+  var EventTracker = ServiceCell.extend({ 
```

Comment:
@kentmw that would only happen if Events is also posting all events that are generated by the view.